ansible_collections/arista/cvp/plugins/modules/cv_tag_v3.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+#
+# GNU General Public License v3.0+
+#
+# Copyright 2019 Arista Networks AS-EMEA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Required by Ansible and CVP
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: cv_tag_v3
+version_added: "4.0.0"
+author: PM Team (@aristanetworks)
+short_description: Create/Assign/Delete/Unassign tags on CVP
+description:
+  - CloudVison Portal Tag module to Create/Assign/Delete/Unassign tags on CloudVision
+options:
+  tags:
+    description: CVP tags
+    required: True
+    type: dict
```

Comment:
This is list not dict.
```suggestion
    type: list
```